mentoring-app/src/app/services/auth.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import { LoginData } from '../models/user.model';
```

Comment:
Try to organize your imports in some way (e.g. first built-in angular imports, then external libraries, then your custom imports or in alphabetical order or in any other appropriate way).

Reply:
done